src/events/events.service.spec.ts

```diff
@@ -513,6 +527,20 @@ describe('EventsService', () => {
         });
 
         await eventsService.create({ type, userId: user.id, points: 100 });
+        await eventsJobsController.updateLatestPoints({
```

Comment:
To avoid importing the worker controller in this test

```suggestion
        await eventsService.updateLatestPoints(user.id, type)
```

Reply:
I think it makes more sense to do `eventsJobsController.updateLatestPoints` because that is the actual function that is being called when the job is executed. Yes, ideally we would test the job separately. Then we would remove ALL of these checks for points in the current tests and just check that the job was kicked off. But that seemed like too much effort and more likelihood of making a mistake / missing some case when copying over all the expectations to the job test.

Reply:
Sort of like in each test
1. Check that the job was kicked off
2. Run the job synchronously and check expectations

If I could've found a way to wait for the asynchronous job to finish and then test expectations that would have been ideal. But I couldn't figure out how to do that

Reply:
Yeah, I agree the ideal scenario will be removing the checks for the points and seeing the job was enqueued in the service since we're moving to eventual consistency. The job tests can test the point value changes. I'm fine leaving as is if you want to keep that refactoring out of this PR.

Reply:
👍🏻 thanks. yea not ideal to not do all the test refactoring but I think it's probably good to get this out quickly to improve API performance